src/modules/home/components/property/property-card.tsx

```diff
@@ -13,17 +15,48 @@ import 'swiper/css/pagination';
 import './swiper.css';
 
 import { EffectFade, Navigation, Pagination } from 'swiper/modules';
+import { axiosInstance } from '#/configs';
+import { useWishlistedProperties } from '../../api/get-wishlisted-properties';
 
-export function ApartmentCard(apartment: Accommodation) {
-  const handleFavoriteClick = (event: React.MouseEvent) => {
+export function ApartmentCard({ id, name, location, pricePerNight, images }: Accommodation) {
+  const { data } = useWishlistedProperties();
+  const navidate = useNavigate();
+  const [isLiked, setIsLiked] = useState(false);
+
+  useEffect(() => {
+    if (data?.data) {
+      const likedFromData = data.data.find((item) => item.id === id);
+      setIsLiked(!!likedFromData);
```

Comment:
Actually, it is the backend's responsibility to check if an accommodation is on the wishlist or not and include a property to indicate it. So you do not have to check it like this.